docs/installation/expansion.rst

```diff
@@ -159,8 +162,10 @@ A node exclusively in the control-plane with ``etcd`` storage will have:
      - effect: NoSchedule
        key: node-role.kubernetes.io/etcd
 
-A worker node dedicated to ``infra`` services (see :doc:`./introduction`) will
-use:
+- A worker node dedicated to ``infra`` services (see :doc:`./introduction`)
+
+  Roles and taints both are set to infra. It's the same as the checkbox of
```

Comment:
roles and taints both are set to infra. It's the same as infra checkbox in MetalK8s GUI

Reply:
Combine the suggestion from you and Lucie I would like to rephrase the sentence in this way:

`roles and taints both are set to infra. It has the same behavior as the **Infra** checkbox in the GUI.`

Is that make sense of you?